### PR TITLE
Updating Tag usage in SQL cmdlets

### PR DESCRIPTION
**Pull request by akromm-zz**
Updating Tag usage in SQL cmdlets

Converting the Tags parameter in SQL cmdlets to use the Azure built Tags helper classes and follow the standard.

### CODE REVIEW

**Comment by cormacpayne** [comment]
@akromm are there any test changes required for the parameter change? Was this part ever covered with tests?

**Comment by akromm** [comment]
@cormacpayne I ran all tests and non required any updates.

File: src/ResourceManager/Sql/Commands.Sql/Server/Cmdlet/SetAzureSqlServer.cs [hunk 1]
```diff
@@ -50,8 +52,8 @@ public class SetAzureSqlServer : AzureSqlServerCmdletBase
         /// </summary>
         [Parameter(Mandatory = false,
             HelpMessage = "The tags to associate with the server.")]
-        [ValidateNotNull]
-        public Dictionary<string, string> Tags { get; set; }
+        [Alias("Tag")]
```
Comment (hovsepm) [inline src/ResourceManager/Sql/Commands.Sql/Server/Cmdlet/SetAzureSqlServer.cs]: What is the value of having Tag alias over property with name "Tag"?

Comment (akromm) [inline src/ResourceManager/Sql/Commands.Sql/Server/Cmdlet/SetAzureSqlServer.cs]: the property name is "Tags" not "Tag".

**Pull request merged**